### PR TITLE
Partners route and db

### PR DESCRIPTION
**Pull request by GeorgesFORTRY**

### CODE REVIEW

**Review by Hacene010 — APPROVED** [review]
It's okey for me

File: index.js [hunk 1]
```diff
@@ -67,6 +67,16 @@ app.get('/members', async (req, res) => {
   });
 });
 
+app.get('/partners', async (req, res) => {
+  connection.query('SELECT * FROM partners', (err, rows) => {
```
Comment (LoicBrassart) [inline index.js]: Either you use callbacks, or you use `async/await`, but not both

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Plz fix

**Review by LoicBrassart — APPROVED** [review]
NmE!

**Pull request merged**